Jenkinsfiles/utils.groovy

```diff
@@ -144,8 +144,9 @@ def announce_push() {
 
 def compose_test() {
     def dc = 'docker-compose -f docker-compose.yml -f docker-compose.test.yml'
+    def dc_down = "${dc} down --volumes --remove-orphans"
```

Comment:
Is it possible to set environment variable `COMPOSE_FILE=docker-compose.yml:docker-compose.test.yml` above, so that you can just call `dc_down=docker-compose down --volumes --remove-orphans`

Reply:
That's an interesting idea, but should be in a new PR.